gravitee-apim-integration-tests/src/test/resources/proto/helloworld.proto

```diff
@@ -1,4 +1,5 @@
-// Copyright 2015 The gRPC Authors
+//
+// Copyright © 2015 The Gravitee team (http://gravitee.io)
```

Comment:
I don't think we should change this copyright 🙈

Reply:
I should have not commit them but each clean install on master is modifying those proto files...